### PR TITLE
Update TR-FC1-ULAKBIM.yaml

### PR DESCRIPTION
**Pull request by hbayindir**
Add Agilia/Articonf to the map.

# Summary

Add Agilia/Articonf to the map for provisioning.

---

**Related issue :**

### CODE REVIEW

File: sites/TR-FC1-ULAKBIM.yaml [hunk 1]
```diff
@@ -28,3 +28,6 @@ vos:
 - auth:
     project_id: a59bd0a9ed754941962846c4a081d6ca
   name: vo.enes.org
+  - auth:
```
Comment (gwarf) [inline sites/TR-FC1-ULAKBIM.yaml:31]:
```suggestion
- auth:
```

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, it seems that indentation is not correct, see the suggested change.

File: sites/TR-FC1-ULAKBIM.yaml (hunk 1 above)
Comment (hbayindir) [reply sites/TR-FC1-ULAKBIM.yaml]: Sorry, my bad. Seems I overlooked while writing it in. Thanks for spotting it.

**Review by enolfc — APPROVED** [review]
Thanks @hbayindir !

**Review by gwarf — APPROVED** [review]

**Pull request merged**